libs/auth/src/artifact_authentication.ts

```diff
@@ -296,18 +295,26 @@ async function performArtifactSpecificAuthenticationChecks(
 ): Promise<void> {
   switch (artifact.type) {
     case 'cast_vote_records': {
-      const metadataFileContents = (
-        await fs.readFile(path.join(artifact.directoryPath, 'metadata.json'))
-      ).toString('utf-8');
-      const metadata: CastVoteRecordExportMetadata = unsafeParse(
-        CastVoteRecordExportMetadataSchema,
-        metadataFileContents
+      const metadataFileContents = await fs.readFile(
+        path.join(artifact.directoryPath, 'metadata.json'),
+        'utf-8'
       );
+      const parseResult = safeParseJson(
```

Comment:
As noted in the commit description, the mistake here was that `unsafeParse` expects an object, not a JSON string. `safeParseJson`, on the other hand, expects a JSON string, which is what we're providing.

Tests missed this because of how I'd incorrectly mocked `unsafeParse` 🙈

I recognize that I should just not mock, but here's why I mocked for context:

```
// Avoid having to prepare a complete CVR.CastVoteRecordReport object for
// CastVoteRecordExportMetadata
```